dhcpd/entrypoint.sh

```diff
@@ -64,7 +64,7 @@ if [ -n "$IFACE" ]; then
         chown dhcpd:dhcpd "$data_dir/dhcpd.leases~"
     fi
 
-    container_id=$(grep docker /proc/self/cgroup | sort -n | head -n 1 | cut -d: -f3 | cut -d/ -f3)
+    container_id=$(grep "docker\|kubepods" /proc/self/cgroup | sort -n | head -n 1 | cut -d: -f3 | cut -d/ -f3)
     if perl -e '($id,$name)=@ARGV;$short=substr $id,0,length $name;exit 1 if $name ne $short;exit 0' $container_id $HOSTNAME; then
         echo "You must add the 'docker run' option '--net=host' if you want to provide DHCP service to the host network."
```

Comment:
Would it make sense to modify this sentence to be a little more generic to apply to both docker or kubepods?

Reply:
Sorry for the pull request re-review submission...  I had my 5-year-old bug me while trying to update the request.  Working on the update...

Reply:
Also, Github seems to be having issues at the moment...  Can you verify that you can see the changes?  I'll resubmit another pull request with them, but the UI seems a little glitchy at the moment.